Source/InstrumentEditorVRC7.cpp

```diff
@@ -225,52 +331,54 @@ void CInstrumentEditorVRC7::LoadInternalPatch(int Num)
 	GetDlgItem(IDC_PASTE)->EnableWindow(FALSE);
 
 	// Register 0
-	Reg = default_inst[(Num * 16) + 0];
+	Reg = default_inst[m_iPatchTone][(Num * 8) + 0];
```

Comment:
personally i don't like the duplicated indexing. maybe try something like `const unsigned char *inst_params = &default_inst[m_iPatchTone][Num * 8]`, and then index into that.